sapl/parlamentares/forms.py

```diff
@@ -216,6 +216,17 @@ class Meta:
             'biografia': forms.Textarea(
                 attrs={'id': 'texto-rico'})}
 
+    def save(self, commit=True):
+        parlamentar = super().save(commit=False)
+        autor = parlamentar.autor.first()
+        usuario = autor.user if parlamentar.autor else None
+
+        if parlamentar and autor and usuario:
```

Comment:
Eu **acho** que o parlamentar sempre vai existir se chegou até aqui... 
```suggestion
        if autor and usuario:
```